packages/integration-sdk-runtime/src/storage/FileSystemGraphObjectStore/FileSystemGraphObjectStore.ts

```diff
@@ -210,19 +210,21 @@ export class FileSystemGraphObjectStore implements GraphObjectStore {
   async flush(
     onEntitiesFlushed?: (entities: Entity[]) => Promise<void>,
     onRelationshipsFlushed?: (relationships: Relationship[]) => Promise<void>,
+    stepId?: string,
   ) {
     await Promise.all([
-      this.flushEntitiesToDisk(onEntitiesFlushed),
-      this.flushRelationshipsToDisk(onRelationshipsFlushed),
+      this.flushEntitiesToDisk(onEntitiesFlushed, stepId),
+      this.flushRelationshipsToDisk(onRelationshipsFlushed, stepId),
     ]);
   }
 
   async flushEntitiesToDisk(
     onEntitiesFlushed?: (entities: Entity[]) => Promise<void>,
+    onlyThisStepId?: string,
```

Comment:
Thoughts on renaming this to just `stepId`? I think it naturally implies that it would be specific to a step. That would be the same verbiage we've used everywhere else.

Reply:
works for me